calcom/cal.com
# Remove global app installed button

## What does this PR do?



Allows Jitsi and Huddle be installed and uninstalled like another app. 

Also fixed rendering "Install App" or "Installed" button text on app page.

Fixes #2372 

## Type of change





## How should this be tested?



For both Jitisi and Huddle
- Go to the app store page and click install
- Go to the installed page you should see that it is now installed with the option to disconnect
- Go to an event type and under locations you should see the only options you have installed
- Go back to the installed page and disconnect the video app
- Under an event type that option should no longer be available for a location
